### DISCUSSION TITLE
Welcome to TheAgentCompany Discussions!

### DISCUSSION THREAD

**Discussion by yufansong** [TheAgentCompany/TheAgentCompany#994] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oshulha** [comment]
Hi! Can I ask you a question? How did you add your users to Plane in TheAgentCompany?
I hope you didn’t have to create fake email addresses just to add 17 users. I’m asking because I’d like to add more users myself, and creating fake emails seems like such a hassle for six hundred users

**Reply by yufansong** [reply]
I remember I manually add it in plane, and then back up the data through here: https://github.com/TheAgentCompany/TheAgentCompany/tree/main/servers/plane#how-to-backup . 

I am not sure whether plane provide api to automatically add user batchly.

**Reply by oshulha** [reply]
Thank you. Understood, unfortunately plane doesn't provide api to add users. Manual approach needs verification from email so that is the problem for me(. Hope to make changes in PostgresSQL to add users

**Reply by yufansong** [reply]
I remember when I add fake user, plane didn't require the verification from email. I don't remember the details, but I am sure we didn't use real email when create fake user. Maybe you can check it again.

**Reply by oshulha** [reply]
I don't know how you do that, but the only way I’ve seen it in the Plane UI is: Workspace Settings → Members → Add Member → Enter their email → Send invitation, and then those users have a pending status
<img width="664" height="187" alt="image" src="https://github.com/user-attachments/assets/d2c34450-8928-411b-a7ab-c47ef2134764" />
<img width="1197" height="66" alt="image" src="https://github.com/user-attachments/assets/75f7bea0-d30b-4cd8-a89c-0eca7a3c84c4" />

And I have a question: how can I use the image that I’ve built with the new data for Plane? At the moment, I don’t know where to use it. I’m thinking of just importing the *.tar.gz files to the api-server image and then checking if I’ll see the updates in the UI after relaunching api-server but at the moment I didn't succeed

**Reply by yufansong** [reply]
I am not sure the register logic. 

For plane back up data, please search this link, the api-server image will download this url when launch. You should replace this logic and make a new api-server image.
https://github.com/TheAgentCompany/the-agent-company-backup-data/releases/download/plane-20241031-0351

**Reply by oshulha** [reply]
Omg, I did it! The hacky solution with Postgres works. I simply added users to the wrong workspace at first — the correct one has the following details:
Workspace Name: TheAgentCompany
Slug: tac
UUID (ID): a56ce149-8ef9-4401-8de6-4dac6a21b4b1

And that moment you mentioned about changing the logic with the link — that’s exactly what I would’ve never noticed. Thanks a lot!!!
The link is placed in restore.sh, which can be downloaded using make download-setup in the servers/plane directory.

Maybe someone else will find this [information](https://www.notion.so/Plane-Users-Adding-Issue-Solution-PUBLIC-28c5e189dacc805ba0a4e2c9f50fe741?source=copy_link) useful
